sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/Transforms.java

```diff
@@ -297,4 +305,73 @@ private static FileReadHeaders toPathReadHeaders(BlobDownloadHeaders h) {
             .setContentCrc64(h.getContentCrc64())
             .setErrorCode(h.getErrorCode());
     }
+
+    static List<BlobSignedIdentifier> toBlobIdentifierList(List<DataLakeSignedIdentifier> identifiers) {
+        if (identifiers == null) {
+            return null;
+        }
+        List<BlobSignedIdentifier> blobIdentifiers = new ArrayList<>();
+        for (DataLakeSignedIdentifier identifier : identifiers) {
+            blobIdentifiers.add(Transforms.toBlobIdentifier(identifier));
+        }
+        return blobIdentifiers;
+    }
+
+    private static BlobSignedIdentifier toBlobIdentifier(DataLakeSignedIdentifier identifier) {
+        if (identifier == null) {
+            return null;
+        }
+        return new BlobSignedIdentifier()
+            .setId(identifier.getId())
+            .setAccessPolicy(Transforms.toBlobAccessPolicy(identifier.getAccessPolicy()));
+    }
+
+    private static BlobAccessPolicy toBlobAccessPolicy(DataLakeAccessPolicy accessPolicy) {
+        if (accessPolicy == null) {
+            return null;
+        }
+        return new BlobAccessPolicy()
+            .setExpiresOn(accessPolicy.getExpiresOn())
+            .setStartsOn(accessPolicy.getStartsOn())
+            .setPermissions(accessPolicy.getPermissions());
+    }
+
+    static FileSystemAccessPolicies toFileSystemAccessPolicies(BlobContainerAccessPolicies accessPolicies) {
+        if (accessPolicies == null) {
+            return null;
+        }
+        return new FileSystemAccessPolicies(Transforms.toDataLakePublicAccessType(accessPolicies.getBlobAccessType()),
+            Transforms.toDataLakeIdentifierList(accessPolicies.getIdentifiers()));
+    }
+
+    static List<DataLakeSignedIdentifier> toDataLakeIdentifierList(List<BlobSignedIdentifier> identifiers) {
+        if (identifiers == null) {
+            return null;
+        }
+        List<DataLakeSignedIdentifier> dataLakeIdentifiers = new ArrayList<>();
+        for (BlobSignedIdentifier identifier : identifiers) {
+            dataLakeIdentifiers.add(Transforms.toDataLakeIdentifier(identifier));
+        }
+        return dataLakeIdentifiers;
+    }
+
+    private static DataLakeSignedIdentifier toDataLakeIdentifier(BlobSignedIdentifier identifier) {
```

Comment:
Same comments as the other Identifier conversion

Reply:
Same as above